identify/README.md

```diff
@@ -3,7 +3,16 @@
 The identify protocol is used to exchange basic information with other peers
 in the network, including addresses, public keys, and capabilities.
 
-There are two variations of the identify protocol, `identify` and `identify/push`.
+There are three variations of the identify protocol, listed below:
+
+| Protocol         | Protocol Id           | Description                                    |
+|------------------|-----------------------|------------------------------------------------|
+| `identify`       | `/ipfs/id/1.0.0`      | Respond to queries for identifying info.       |
+| `identify/push`  | `/ipfs/id/push/1.0.0` | Send info proactively to inform about changes. |
+| `identify/delta` | `/p2p/id/delta/1.0.0` | Send partial updates to inform about changes.  |
```

Comment:
Is this not just "identify push 1.1.0"?

(either way works for me, this just feels like identify push with an extra feature)